src/widgets/page/LocalPage.tsx

```diff
@@ -8,13 +8,12 @@ import Page from './Page';
 
 export interface LocalPageProperties {
 	path: string;
-	hasLeftSideMenu?: boolean;
 	warpInPage?: boolean;
 }
 
 export default class LocalPage extends WidgetBase<LocalPageProperties> {
 	protected render() {
-		const { path, hasLeftSideMenu = false, warpInPage = true } = this.properties;
+		const { path, warpInPage = true } = this.properties;
```

Comment:
I think this should be `wrapInPage` (although `warpInPage` sounds pretty cool, so carry on if it's intentional ✊)